lms/djangoapps/bulk_email/tasks.py

```diff
@@ -52,6 +52,7 @@
     update_subtask_status
 )
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.lib.celery import APP
```

Comment:
Hmm, read the docstring in `openedx.core.lib.celery`.  I think we want to use the `shared_task` decorator instead, see the sidebar under https://docs.celeryproject.org/en/stable/userguide/tasks.html#basics .  Same for the other files below.